packages/frontend/src/components/VisualizationConfigs/common/AddButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button, type ButtonProps } from '@mantine/core';
+import { type ButtonHTMLAttributes, type FC } from 'react';
+
+type Props = ButtonProps & ButtonHTMLAttributes<HTMLButtonElement>;
+
+export const AddButton: FC<Props> = ({ ...props }) => (
+    <Button
+        size="sm"
+        variant="subtle"
+        compact
+        leftIcon="+"
```

Comment:
no real icon 😢

Reply:
yup! I removed it because of the padding to match @PriPatel's design requirements :D